src/platforms/android/androidutils.h

```diff
@@ -32,6 +33,14 @@ class AndroidUtils final : public QObject {
 
   static void appReviewRequested();
 
+  static void dispatchToMainThread(std::function<void()> callback);
+
+  static QByteArray getQByteArrayFromJString(JNIEnv* env, jstring data);
```

Comment:
About the compilation issue, it seems that `androidsharedprefs.cpp` doesn't need to include `androidutils.h`. If you remove the include, the unit-tests should compile.

Reply:
About the unit-test you also need to add `void MozillaVPN::launchPlayStore() {}` in the 2 `mocmozillavpn.cpp` files in the `tests` folder.

Reply:
Fixing the `androidsharedprefs.cpp` #include should fix the wasm compilation too.